lib/middleware/routes-output/routes-output.js

```diff
@@ -96,7 +82,16 @@ const generateOutput = async (req, res) => {
     let output = await nunjucksConfiguration.renderPage(pageInstance, Object.assign({}, res.locals)) // getPageOutput()
     output = output.replace(/<head>/, `<head><base href="http://localhost:${PORT}">`)
     output = output.replace(/<title>.*<\/title>/, `<title>${submission}</title>`)
-    // const context = await browser.createIncognitoBrowserContext()
+
+    const browser = await puppeteer.launch({
+      args: [
+        '--no-sandbox',
+        '--disable-setuid-sandbox',
+        '--disable-dev-shm-usage'
+      ]
+    })
+
+    const browserProcessID = browser.process().pid
     let page
```

Comment:
Is it worth putting a try/catch around the page instantiation/render/close parts and killing the browser in the catch block as well. Otherwise, if the page blows up the browser instance will still hang around, no?

Reply:
Have wrapped all the browser methods in one try catch now